web/templates/popovers/left_sidebar/navigation_view_hide_popover.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="popover-menu" data-simplebar data-simplebar-tab-index="-1">
+    <ul role="menu" class="popover-menu-list">
+        <li role="none" class="link-item popover-menu-list-item">
+            {{#if is_currently_active_unpinned_view}}
+            <a role="menuitem" class="show-navigation-view popover-menu-link" data-fragment="{{fragment}}" tabindex="0">
+                <i class="popover-menu-icon zulip-icon zulip-icon-eye" aria-hidden="true"></i>
+                <span class="popover-menu-label"><tr>Show <b>{{view_name}}</b> in left sidebar</tr></span>
```

Comment:
These all need to be properly tagged for translation. `<tr>` is a table row, not a translation tag :).